dist/obs-server.spec

```diff
@@ -567,6 +567,15 @@ getent passwd obsapidelayed >/dev/null || \
   /usr/sbin/useradd -r -s /bin/bash -c "User for build service api delayed jobs" -d /srv/www/obs/api -g www obsapidelayed
 %service_add_pre %{obs_api_support_scripts}
 
+# On upgrade keep the values for the %post script
+if [ "$1" == 2 ]; then
+  systemctl --quiet is-enabled obsapidelayed.service && touch %{_rundir}/enable_obs-api-support.target
+  if systemctl --quiet is-active  obsapidelayed.service;then
+    touch %{_rundir}/start_obs-api-support.target
+    systemctl stop               obsapidelayed.service
```

Comment:
shouldnt you also disable it?

Reply:
Shouldn't this be done automatically if the service is removed?

Reply:
uhm dont think so. we normally only remove services on uninstall.

Reply:
done